js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx

```diff
@@ -42,6 +42,10 @@ const labelForBackfillStatus = (key: BulkActionStatus) => {
       return 'Failed';
     case BulkActionStatus.REQUESTED:
       return 'In progress';
+    case BulkActionStatus.COMPLETED_SUCCESS:
```

Comment:
I matched the tense of the existing labels for this PR, but the runs page uses `success` and `failure` instead of `succeeded` and `failed`. should i align them? 

cc @salazarm

Reply:
maybe not so important for this PR, but we should just keep it in mind to make sure the runs feed uses consistent terminology

Reply:
The tenses within `DagsterRunStatus` and step events are also annoyingly inconsistent with each other.  E.g. STARTED vs. FAILURE.  However I think we use SUCCESS pretty consistently throughout the product, so I support using that instead of SUCCEEDED.